recipe/meta.yaml

```
@@ -1,51 +1,62 @@
 {% set name = "peewee" %}
-{% set version = "3.15.4" %}
+{% set version = "3.18.2" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 2581520c8dfbacd9d580c2719ae259f0637a9e46eda47dfc0ce01864c6366205
+  - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 77a54263eb61aff2ea72f63d2eeb91b140c25c1884148e28e4c0f7c4f64996a0
+  - url: https://github.com/coleifer/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 20f5ca85d46f0c251cba5ab6f734b09e89ea77af56ad66708225bc7d4331f4c7
+    folder: gh
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
+    - python
     - {{ compiler('c') }}
   host:
     - python
     - pip
-    - cython >=0.22.1
-    - msinttypes  # [win]
+    - setuptools
+    - wheel
+    - cython
     - sqlite
```

There's an entry in `cbc.yaml` so we can constrain this in `host`:

```suggestion
    - sqlite {{ sqlite }}
```